controllers/admin/AdminMetaController.php

```diff
@@ -484,7 +484,7 @@ public function generateRobotsFile()
                 fwrite($write_fd, "# Files\n");
                 foreach ($this->rb_data['Files'] as $iso_code => $files) {
                     foreach ($files as $file) {
-                        if (!empty($language_ids)) {
+                        if (!empty($language_ids) && count($language_ids) > 1) {
```

Comment:
It's for multilanguage.

Multi : 
-Disallow: /*fr/recuperation-mot-de-passe
-Disallow: /*fr/recherche
-Disallow: /*en/password
-Disallow: /*en/search


Not multi : 
Disallow: /panier
Disallow: /recherche